cves/CVE-2020-7471.yml

```diff
@@ -103,10 +103,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: This would be hard to catch in unit tests, since if you could predict the ways your sanitizer could break, you'd just make a better sanitizer.
```

Comment:
Maybe testing from an outside source would be more helpful in this instance.

Reply:
I don't really like testing from outside sources, but it would be a valid method of testing. It still wouldn't be guaranteed to catch it though